scripts/update-lobster.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# this script pulls the evergreen-ci/lobster repo and copies the files so that
+# they can be served by evergreen
+# it should only be run from the base evergreen directory
+if [ -n "$LOBSTER_TEMP_DIR" ] && [ -n "$LOBSTER_TEMP_DIR" ]; then
+  set -o errexit
+  set -o xtrace
+
+  echo $LOBSTER_TEMP_DIR
+  LOBSTER_REPO=https://github.com/evergreen-ci/lobster.git
+  LOBSTER_TEMP_ASSETS_DIR="$LOBSTER_TEMP_DIR/build"
+  LOBSTER_STATIC_DIR="$LOBSTER_TEMP_ASSETS_DIR/static"
+  LOBSTER_HTML="$LOBSTER_TEMP_ASSETS_DIR/index.html"
+  EVG_LOB_MANIFEST_DIR="$EVGHOME/public/static/lobster"
+  EVG_LOB_STATIC_DIR="$EVG_LOB_MANIFEST_DIR/static"
```

Comment:
nit: can we consistently call the variables "LOBSTER" or "LOB" rather than using both interchangeably? Same for EVERGREEN - either use EVG or EVERGREEN but not both.